index.html

```diff
@@ -960,6 +960,19 @@ <h3>Controller Documents</h3>
 certain <a>verification methods</a> for specific purposes.
         </p>
 
+        <p class="issue atrisk"
+          title="Potential for stand-alone Controller Document specification">
+This section on Controller Documents has been identified as potentially useful
+for other securing mechanisms such as [[?VC-JOSE-COSE]]. There is a possibility
+that this section will be moved to an independent Controller Document
+specification, which would then be normatively referenced by this specification.
```

Comment:
Although historically correct, I think a more balanced message would be better, to avoid raised eyebrows. Something like:

```suggestion
There are lots of commonalities between this section on Controller Documents and similar sections in other securing mechanisms such as [[?VC-JOSE-COSE]], or similar concepts in specifications such as [[?DID-CORE]]. The Working Group is currently discussing the possibility to move this section to an independent Controller Document specification that can be referenced normatively.
```